README.md

```diff
@@ -91,10 +91,10 @@ You can also access the JavaScript variables like this
 ```js
 import { customProperties } from '@commercetools-frontend/ui-kit';
```

Comment:
I think this export should point to the js object, no the json one. I also changed the export in `src/index`